server/src/db/repositories/reportRepository.ts

```diff
@@ -1,7 +1,7 @@
-const ReportModel = require('../schemas/report');
+import ReportModel from '../schemas/report';
 
-const { getReportDetailsPipeline, sortReportDetails } = require('../pipelines/reportDetails');
-const { getLatestReportPipeline, sortLatestReports, branchReports } = require('../pipelines/latestReports');
+import { getReportDetailsPipeline, sortReportDetails } from '../pipelines/reportDetails';
+import { getLatestReportPipeline, sortLatestReports, branchReports } from '../pipelines/latestReports';
 
 function addSortingToPipeline(sorting, aggregation, sortFunction) {
```

Comment:
I think we should add types for this input as well.

Reply:
done